### PR TITLE
translations: restore and improve translations

### PR DESCRIPTION
**Pull request by iGormilhit**
* Updates transifex config according to the new resource slug.
* Ignores webpack assets when extracting strings from JS and JSON files.
* Ignores compiled documents JSON schemas when extracting strings from
  JSON files.
* Extracts new strings and updates catalogs.
* Prefixes language codes, country codes and canton codes to avoid the
  same code beeing used multiple times.
* Restores language codes, country codes and canton codes translations.
* Changes TVA to VAT.
* Fixes some transifex issues, like the interface switch label, or some
  statements in the document JSON schema.
* Corrects a description in the document JSON schema file.
* Updates tests with lang_ prefixes.

Co-Authored-by: Igor Milhit <igor.milhit@rero.ch>
Co-Authored-by: Alicia Zangger <alicia.zangger@rero.ch>

## Why are you opening this PR?

- To fix issues opened on transifex: https://www.transifex.com/rero/reroils/translate/#issues.
- :warning: this PR has a lot of extracted or modified string from previous commits.

## How to test?

- Extracting string should avoid `invenio-assets`.
- Check translations in the files.

## Code review check list

- [ ] Commit message template compliance.
- [ ] Commit message without typos.
- [ ] File names.
- [ ] Functions names.
- [ ] Functions docstrings.
- [ ] Unnecessary commited files?

### CODE REVIEW

File: rero_ils/translations/de/LC_MESSAGES/messages.po [hunk 1]
```diff
@@ -4,22 +4,22 @@
...
+# Nicolas Prongué <n.prongue@outlook.com>, 2019
 # alicia.zangger <alicia.zangger@gmail.com>, 2019
 # Renaud Michotte <renaud.michotte@gmail.com>, 2019
-# Johnny Mariéthoz <johnny.mariethoz@rero.ch>, 2019
-# Peter Weber <peter.weber@rero.ch>, 2019
-# Nicolas Prongué <n.prongue@outlook.com>, 2019
 msgid ""
 msgstr ""
-"Project-Id-Version: rero-ils 0.5.2\n"
+"Project-Id-Version: rero-ils 0.5.0\n"
```
Comment (blankoworld) [inline rero_ils/translations/de/LC_MESSAGES/messages.po]: Is there any reason to retrograde the rero-ils version here?

File: rero_ils/translations/fr/LC_MESSAGES/messages.po [hunk 2]
```diff
@@ -1016,10171 +930,5099 @@ msgstr ""
...
-#: rero_ils/modules/documents/jsonschemas/documents/document-minimal-v0.0.1_src.json:1503
-#: rero_ils/modules/documents/jsonschemas/documents/document-v0.0.1.json:779
-#: rero_ils/modules/documents/jsonschemas/documents/document-v0.0.1.json:3222
-#: rero_ils/modules/documents/jsonschemas/documents/document-v0.0.1.json:8882
-#: rero_ils/modules/documents/jsonschemas/documents/document-v0.0.1_src.json:1503
-msgid "ast"
-msgstr "asturien"
+#: rero_ils/modules/documents/jsonschemas/documents/document-minimal-v0.0.1_src.json:459
+#: rero_ils/modules/documents/jsonschemas/documents/document-v0.0.1_src.json:459
+msgid "Statement of place and agent of the provision activity."
```
Comment (AoNoOokami) [inline rero_ils/translations/fr/LC_MESSAGES/messages.po]: Not sure I remember well, but shouldn't this be replaced by "Statement"?

**Comment by iGormilhit** [comment]
> All language translations missing in po's?

@rerowep Not anymore relevant.

File: rero_ils/translations/fr/LC_MESSAGES/messages.po (hunk 2 above)
Comment (iGormilhit) [reply rero_ils/translations/fr/LC_MESSAGES/messages.po]: Yes. Done now.

**Pull request merged**